platform-api/che-core-api-infrastructure-local/src/main/java/org/eclipse/che/api/local/LocalInfrastructureModule.java

```diff
@@ -45,6 +46,7 @@ protected void configure() {
 //        bind(FactoryStore.class).to(InMemoryFactoryStore.class);
```

Comment:
Remove commented code if you've changed this class. We don't need to have commented code in our sources.